src/__tests__/incidents/list/ViewIncidentsTable.test.tsx

```diff
@@ -1,94 +1,77 @@
 import IncidentRepository from '../../../shared/db/IncidentRepository'
 import Incident from '../../../shared/model/Incident'
+import { extractUsername } from '../../../shared/util/extractUsername'
 
 describe('View Incidents Table', () => {
-  const setup = async (
+  const expectedIncident = {
+    id: 'incidentId1',
+    code: 'someCode',
+    date: new Date(2020, 7, 4, 0, 0, 0, 0).toISOString(),
+    reportedOn: new Date(2020, 8, 4, 0, 0, 0, 0).toISOString(),
+    reportedBy: 'com.test:user',
+    status: 'reported',
+  } as Incident
+
+  const setup = (
     expectedSearchRequest: IncidentSearchRequest,
-    expectedIncidents: Incident[],
+    expectedIncidents = [expectedIncident],
   ) => {
     jest.spyOn(IncidentRepository, 'search').mockResolvedValue(expectedIncidents)
-
-    let wrapper: any
     const history = createMemoryHistory()
-    await act(async () => {
-      wrapper = await mount(
+
+    return {
+      history,
+      ...render(
         <Router history={history}>
           <ViewIncidentsTable searchRequest={expectedSearchRequest} />
         </Router>,
-      )
-    })
-    wrapper.update()
-
-    return { wrapper: wrapper as ReactWrapper, history }
+      ),
+    }
   }
 
   beforeEach(() => {
     jest.resetAllMocks()
   })
 
-  it('should call the incidents search with the search request', async () => {
-    const expectedSearchRequest: IncidentSearchRequest = {
-      status: IncidentFilter.all,
-    }
-    await setup(expectedSearchRequest, [])
-
-    expect(IncidentRepository.search).toHaveBeenCalledTimes(1)
-    expect(IncidentRepository.search).toHaveBeenCalledWith(expectedSearchRequest)
-  })
-
   it('should display a table of incidents', async () => {
-    const expectedIncidents: Incident[] = [
-      {
-        id: 'incidentId1',
-        code: 'someCode',
-        date: new Date(2020, 7, 4, 0, 0, 0, 0).toISOString(),
-        reportedOn: new Date(2020, 8, 4, 0, 0, 0, 0).toISOString(),
-        reportedBy: 'com.test:user',
-        status: 'reported',
-      } as Incident,
-    ]
-    const { wrapper } = await setup({ status: IncidentFilter.all }, expectedIncidents)
-
-    const incidentsTable = wrapper.find(Table)
-
-    expect(incidentsTable.exists()).toBeTruthy()
-    expect(incidentsTable.prop('data')).toEqual(expectedIncidents)
-    expect(incidentsTable.prop('columns')).toEqual([
-      expect.objectContaining({ label: 'incidents.reports.code', key: 'code' }),
-      expect.objectContaining({ label: 'incidents.reports.dateOfIncident', key: 'date' }),
-      expect.objectContaining({ label: 'incidents.reports.reportedBy', key: 'reportedBy' }),
-      expect.objectContaining({ label: 'incidents.reports.reportedOn', key: 'reportedOn' }),
-      expect.objectContaining({ label: 'incidents.reports.status', key: 'status' }),
-    ])
-    expect(incidentsTable.prop('actionsHeaderText')).toEqual('actions.label')
+    setup({ status: IncidentFilter.all })
```

Comment:
I think that we need to add some `expect` statements here to include what `expect(incidentsTable.prop('data')).toEqual(expectedIncidents)` was testing.

Reply:
@emma-r-slight you resolve this one too?